src/common/transformations/tests/transpose_sinking/ts_gather_test.cpp

```diff
@@ -64,6 +64,7 @@ auto wrapper = [](const TestCase& test_case) {
 struct GatherForwardArguments {
     OutputVector inputs_to_main;
     Output<Node> new_input_to_Gather_1;
+    ov::AxisVector new_transpose_order;
```

Comment:
Avoid using `ov::`, there's `using namespace ov;` at the beginning of the file (applies elsewhere)